packages/discovery-react-components/src/components/TooltipHighlight/TooltipHighlight.tsx

```diff
@@ -21,6 +21,11 @@ type Props = {
   tooltipAction: TooltipAction;
 };
 
+// Longer strings will be truncated with ellipsis in the middle of the term.
+// This way a user sees the start and end of the string and can map it to the document view
+const MAX_CONTENT_LENGTH = 30; // even number
```

Comment:
Why does this have to be an "even number"? If not necessary, just remove that comment.

Reply:
Tested and it works with odd number. I removed the comment.